modules/express/src/lightning/lightningSignerRoutes.ts

```diff
@@ -54,7 +54,19 @@ function getMacaroonRootKey(passphrase: string, nodeAuthEncryptedPrv: string, de
  * Handle the request to initialise remote signer LND for a wallet.
  */
 export async function handleInitLightningWallet(req: express.Request): Promise<unknown> {
-  const { walletId, passphrase, signerTlsKey, signerTlsCert, signerIp, expressIp } = decodeOrElse(
+  const bitgo = req.bitgo;
+  const coinName = req.params.coin;
+  if (!isLightningCoinName(coinName)) {
+    throw new Error(`Invalid coin to initialise lightning wallet: ${coinName}`);
+  }
+  const coin = bitgo.coin(coinName);
+
+  const walletId = req.params.id;
+  if (typeof walletId !== 'string') {
+    throw new Error(`Invalid wallet id: ${walletId}`);
+  }
+
+  const { passphrase, signerTlsKey, signerTlsCert, signerIp, expressIp } = decodeOrElse(
```

Comment:
overall, the way we should go through the routes is
1. get and validate coin
2. get walletId
3. parse the params